meshtastic/node.py

```
@@ -195,9 +195,40 @@ def _signal_metadata_stdout_event(self) -> None:
         if metadata_stdout_event is not None:
             metadata_stdout_event.set()
 
+    def _get_metadata_snapshot(self) -> Any:
```

Ruff (0.15.5)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_get_metadata_snapshot`

(ANN401)